doc/jekyll_tools/js/jumbotron.js

```diff
@@ -0,0 +1,276 @@
+var startAnimation = function(){
```

Comment:
This JS file is  the "moving graph" animation in the jumbotron of the main homepage; it is not used in the Python pages as it consumes too much CPU.

Reply:
alright, removed the file. I adapted the layout template a little, leaving the sourcing in a permantly false if statement. That should be a compromise between explaining the logic of the live page and not having too much clutter around.